src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=root
+spring.datasource.password=
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.hibernate.ddl-auto=update
+# Enabling H2 Console
+spring.h2.console.enabled=true
+spring.datasource.initialization-mode=always
```

Comment:
What does this property do? Why did you use it?

Reply:
I'm not sure but it probably give to opportunity to customize spring behavior to create schema. It's not nessery so I'm going to remove it.